### ISSUE TITLE
create a new deno project and compile fails

### ISSUE THREAD

**Issue by eswr** [denoland/deno#15709]
Steps to reproduce:
1. create a new deno project
2. compile (`deno compile -A -c deno.json main.ts`)

When I try to compile with `deno compile -A -c deno.json main.ts`, I get this error:
```
============================================================
Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: macos aarch64
Version: 1.25.0
Args: ["deno", "compile", "-A", "-c", "deno.json", "main.ts"]

thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /Users/ib/.cargo/registry/src/github.com-1ecc6299db9ec823/eszip-0.25.0/src/v2.rs:512:60
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

info:
`deno --version`
```
deno 1.25.0 (release, aarch64-apple-darwin)
v8 10.6.194.5
typescript 4.7.4
```

**Comment by eswr** [comment]
I tried to deploy it via GitHub actions, I got the following error:
```
Error: The deployment failed: Relative import path "$fresh/server.ts" not prefixed with / or ./ or ../
```

**Comment by bartlomieju** [comment]
I believe this was already fixed in https://github.com/denoland/deno/pull/15672 and will be released in v1.25.1 tomorrow.
